src/styles/blocks/catalog.scss

```diff
@@ -11,15 +11,22 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    position: relative;
 
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $transition-time ease;
```

Comment:
it is more correct to enumerate the properties for which it will be applied transition:
`transition: transform $transition-time ease, color $transition-time ease`

Reply:
I've fixed it, thanks)